01_get_Export.R

```diff
@@ -9,6 +9,7 @@ list_of_problems <- list()
 # for each file in the csv, loop through the file names in the csv
 for (file in unique(cols_and_data_types$File)) {
   #import the csv and save it as a data frame
+  print(paste0("importing ", file))
```

Comment:
@kiadso - this is just so we have better logging of steps. may help us debug issues more quickly

Reply:
I really like this! thank you!